man/scale_hue.Rd

```diff
@@ -14,7 +14,7 @@ scale_colour_hue(
   h.start = 0,
   direction = 1,
   na.value = "grey50",
-  aesthetics = "colour"
+  aesthetics = aesthetic
```

Comment:
You can avoid this usage parameter to be misreported by explicitly setting the formal.

For example, the hue factory could be:

```r
scale_colour_hue_factory <- function(aesthetic) {
  fun <- function(..., h = c(0, 360) + 15, c = 100, l = 65, h.start = 0,
           direction = 1, na.value = "grey50") {
    discrete_scale(aesthetics, palette = hue_pal(h, c, l, h.start, direction),
                   na.value = na.value, ...)
  }
  formals(fun)$aesthetics <- aesthetic
  fun
}
```